ydb/docs/en/core/reference/ydb-sdk/topic.md

```diff
@@ -1451,6 +1455,8 @@ If a commit fails with an error, the application should log it and continue; it
    }
    ```
 
+   The `Commit` call is fast by default, saving data into an internal buffer and returning control back to the caller. The real message to the server is sent in the background. To prevent losing the last commits, you should call the `Reader.Close()` method before exiting the program.
```

Comment:
We'd better make it an `{% include %}` file instead of copypaste.

Reply:
It has two paragraph and two copy of each in one file. I need two additional files and four include options for deduplicate of them. The paragraph doesn't need in other files and replace the text to include option make read the text difficult when read it.

I think about small copy-paste better, then a lot of small files and combine them by includes.